pandas/core/arrays/base.py

```diff
@@ -932,6 +977,26 @@ def _formatting_values(self) -> np.ndarray:
     # Reshaping
     # ------------------------------------------------------------------------
 
+    def reshape(self, *shape):
+        """
+        Return a view on this array with the given shape.
```

Comment:
Would be good to document that only `(1, N)` and `(N, 1)` is allowed here.

Reply:
Good thinking.  ATM I wrote most of this with only (1, N) in mind